builtin/game/misc.lua

```diff
@@ -197,3 +197,16 @@ function core.http_add_fetch(httpenv)
 
 	return httpenv
 end
+
+local ESCAPE_CHAR = string.char(0x1b)
+function core.get_color_escape_sequence(color)
+	return ESCAPE_CHAR .. "(color@" .. color .. ")"
```

Comment:
Wouldn't it be better to have a shorter color descriptor? Like "c@" instead of "color@" ?